up-l1/zenoh.adoc

```diff
@@ -35,9 +35,138 @@ Zero Overhead Network Protocol.
 
 Zenoh /zeno/ is a pub/sub/query protocol unifying data in motion, data at rest and computations. It elegantly blends traditional pub/sub with geo distributed storage, queries and computations, while retaining a level of time and space efficiency that is well beyond any of the mainstream stacks.
 
-
-for more information, please visit https://projects.eclipse.org/projects/iot.zenoh or https://zenoh.io/
+For more information, please visit https://projects.eclipse.org/projects/iot.zenoh or https://zenoh.io/
 
 == Specifications
 
-_Coming Soon_
\ No newline at end of file
+=== UAttribute Mapping
+
+Zenoh supports user attachment.
+We **MUST** send additional information with the mechanism, for example, UAttribute.
+This reduces the unnecessary serialization on payload, which mostly takes time.
+
+User attachment supports key-value pairs, and the following table is the key-value pairs using now.
+
+[cols="1,1"]
+|===
+| key | value
+
+| "uattr"
+| UAttribute object encoded into protobuf
+|===
+
+Besides the user attachment, some features in the UAttribute **SHOULD** be mapped into Zenoh transport settings.
```

Comment:
This seems rather unspecific: *some* features (which?) *SHOULD* be mapped differently (how?) ...
Or does this refer to the following sections? If so, I guess we could simply remove this sentence as it does not provide any guidance.

Reply:
There is no v2 of the protocol, we have to finish defining and implementing v1 and then only afterwards can we consider breaking it to define v2 :-).

Reply:
Yes, it refers to the following sections. I'll remove the sentence. Thanks!